src/components/operations/operation-details/ko/runtime/type-definition-object.html

```diff
@@ -66,9 +66,7 @@
 
                 <!-- /ko -->
             </div>
-            <!-- ko if: $component.examples -->
-            <div role="cell" class="text-truncate" data-bind="text: $component.examples[property.name], attr: {title: $component.examples[property.name]}"></div>
-            <!-- /ko -->
+            <div role="cell" class="text-truncate" data-bind="text: property.example, attr: {title: property.example}"></div>
```

Comment:
why was this removed?

Reply:
It is not removed, it is fixed. The previous logic was weird - values were taken from the big example by field name, now I am taking examples straight for the property.

Reply:
but (from the code) if examples are not defined, we are rendering a div, while previously we were not at all, right?
I didn't check it rendered, I'm just thinking that it might look weird if there are no examples but if you rested it, it's all good